js/constants.js

```diff
@@ -4,6 +4,6 @@
 'use strict';
 
 module.exports = {
-  EXAMPLE_CONTEXT_URL: 'https://w3id.org/example/v1',
-  EXAMPLE_CBORLD_VALUE: 0x0 // value needs to be > 0x8000
+  CONTEXT_URL: 'https://w3id.org/security/ed25519-signature-2020/v1',
+  CBORLD_VALUE: 0x8001
```

Comment:
I have no idea if this is appropriate or not. It seems like it would clash with other things. We should figure out how we want to deal with this.

Reply:
Yeah, agreed, keeping track is going to be an issue. I literally had to search through all our repos/contexts where we specify those values, to make sure there's no collision.

Reply:
Now registered over at https://github.com/digitalbazaar/cbor-ld-spec/pull/3